records/record-connection.feature

```diff
@@ -44,19 +44,31 @@ Feature: Record Connectivity
       And client B receives at least one "CONNECTION" error "CONNECTION_ERROR"
       And all clients have record "record" with path "user.firstname" and data 'Mike2'
 
-  Scenario: Remote wins by default
-    When server 1 goes down
-      And client A sets the record "record" and path "user.firstname" with data 'Mike'
-      And client B sets the record "record" and path "user.firstname" with data 'Sam'
+  #Flaky test
+  #Scenario: Remote wins by default
+    #When server 1 goes down
+      #And client A sets the record "record" and path "user.firstname" with data 'Mike'
+      #And client B sets the record "record" and path "user.firstname" with data 'Sam'
 
-    Then client A has record "record" with path "user.firstname" and data 'Mike'
-    Then client B has record "record" with path "user.firstname" and data 'Sam'
-    Then client C has record "record" with path "user.firstname" and data 'John'
-    Then client D has record "record" with path "user.firstname" and data 'John'
+    #Then client A has record "record" with path "user.firstname" and data 'Mike'
+    #Then client B has record "record" with path "user.firstname" and data 'Sam'
+    #Then client C has record "record" with path "user.firstname" and data 'John'
+    #Then client D has record "record" with path "user.firstname" and data 'John'
 
-    When server 1 comes back up
+    #When server 1 comes back up
 
-    Then client A receives at least one "CONNECTION" error "CONNECTION_ERROR"
+    #Then client A receives at least one "CONNECTION" error "CONNECTION_ERROR"
+      #And client B receives at least one "CONNECTION" error "CONNECTION_ERROR"
+      #And all clients have record "record" with path "user.firstname" and data 'Mike'
+
+  Scenario: Write acknowledgements called immediately
```

Comment:
is this a duplicate of  ` Scenario: Connection goes down and clients are notified of write failure immediately`?